packages/store/src/module.ts

```diff
@@ -31,29 +33,11 @@ export class NgxsRootModule {
     select: SelectFactory,
     @Optional()
     @Inject(ROOT_STATE_TOKEN)
-    states: any[]
+    states: StateClass[]
```

Comment:
Is this correct?
The state classes might not have a recognised `META_KEY` at this stage (even though the decorator creates it I'm not sure if it is seen in the type information). If this does work then use `StateClass`, if it doesn't then we could use Type<unknown> instead.

Reply:
decorator is called before forRoot, forFeature